easybuild/easyconfigs/t/torchvision/torchvision-0.5.0-fosscuda-2019b-Python-3.7.4-PyTorch-1.4.0.eb

```
@@ -0,0 +1,35 @@
+easyblock = 'PythonPackage'
+
+name = 'torchvision'
+version = '0.5.0'
+local_pytorch_ver = '1.4.0'
+versionsuffix = '-Python-%%(pyver)s-PyTorch-%s' % local_pytorch_ver
+
+homepage = 'https://github.com/pytorch/vision'
+description = " Datasets, Transforms and Models specific to Computer Vision"
+
+toolchain = {'name': 'fosscuda', 'version': '2019b'}
+
+source_urls = ['https://github.com/pytorch/vision/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['eb9afc93df3d174d975ee0914057a9522f5272310b4d56c150b955c287a4d74d']
+
+builddependencies = [('CMake', '3.15.3')]
+
+local_pysuff = '-Python-%(pyver)s'
+dependencies = [
+    ('Python', '3.7.4'),
+    ('Pillow-SIMD', '6.0.x.post0'),
+    ('PyTorch', '1.4.0', local_pysuff),
```

Same here, I suggest to drop the variable `local_pysuff` as it not used anywhere else
```suggestion
dependencies = [
    ('Python', '3.7.4'),
    ('Pillow-SIMD', '6.0.x.post0'),
    ('PyTorch', '1.4.0', '-Python-%(pyver)s'),
```